.github/workflows/arm-auto-signoff-preview.yaml

```diff
@@ -0,0 +1,60 @@
+name: ARM Auto Signoff (Preview)
+
+on:
+  pull_request:
+    types:
+      - labeled
+      - unlabeled
+      - opened
+      - reopened
+      - synchronize
+  workflow_dispatch:
+    inputs:
+      owner:
+        description: The account owner of the repository. The name is not case sensitive.
+        required: true
+        type: string
+      repo:
+        description: The name of the repository without the .git extension. The name is not case sensitive.
+        required: true
+        type: string
+      issue_number:
+        description: The number of the pull request.
+        required: true
+        type: string
+      head_sha:
+        description: The SHA of the commit.
+        required: true
+        type: string
+
+permissions:
+  contents: read
+
+jobs:
+  arm-auto-signoff-preview:
+    name: ARM Auto Signoff (Preview)
+
+    runs-on: ubuntu-24.04
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          # Required to determine files changed in PR
+          fetch-depth: 2
+          # TODO: Temporary to improve perf while iterating
```

Comment:
I assume you want to undo this before merging?

Reply:
Yes, before merging I'll remove the sparse checkout, since the check needs to clone the whole repo to look for changes.